generate_workspace/src/main/java/com/google/devtools/build/workspace/output/BzlWriter.java

```diff
@@ -32,27 +35,33 @@
       MethodHandles.lookup().lookupClass().getName());
 
   private final String[] argv;
-  private final Path generatedFile;
+  private final File generatedFile;
```

Comment:
Solely for unit tests with jimfs. 

The behavior of `createParentDirectory` differed in the in-memory filesystem differed between when I used `new File()` and when I used `somePath.toFile()` With the `new File()`everything (tests and actual tool) performed as desired. Using the Path, the tests would fail to create the directories. 

The only real distinction is that with one, I am performing:

```
createParentDirectory(outputFile);
```
and with the other I am doing
```
File file = new File(ouputDir.toString(), "generate_workspace.bzl");
...
createParentDirectory(file.getAbsoluteFile().toPath());
```